ci/unit/test_main.sh

```diff
@@ -7,6 +7,27 @@ set -ex
 # JENKINS-52781 tar function is breaking symlinks
 rm -rf test_results
 mkdir test_results
+
+# Check if this is a Bulleye stage
+USE_BULLSEYE=
+if [ -n "${STAGE_NAME:?}" ]; then
```

Comment:
You don't need to test this for a value before using in a `case` statement:
```suggestion
```

I.e.:
```
$ case $foo in
> bar) echo "bar"
> ;;
> esac
```
doesn't produce any kind of error.

Reply:
Updated code